lib/fog/compute/kubevirt/models/vms.rb

```diff
@@ -48,7 +48,7 @@ def get(name)
         #       :macAddress => '12:34:56:AB:CD:EF' }
         #   ]
         #
-        # One of :image or :pvc needs to be provided.
+        # One of :image or one or more :pvc(s) needs to be provided.
```

Comment:
so the *pvc* will remain as the name of the attribute, but could get one or many strings representing the PVC names.
In that case, we should document the type of it like
```
  @param [String] image...
  or
  @param[Array]/@param[String] pvc - a single or multiple strings of the claim names
```

But, with this we don't have an option to specify which claim should be the **bootable** disk of the VM.
While this API serves the functionality of adding multiple pvcs, we should a way to specify also properties of the disk device.
But for that, I'll create a volume fog entity to represent that.

Reply:
Will update the docsting